### PR TITLE
Recap of WTD Day AU

### PR DESCRIPTION
**Pull request by swapnilogale**
@ericholscher @plaindocs - Can you review this?

### CODE REVIEW

File: docs/conf/au/2017/index.rst [hunk 1]
```diff
@@ -104,43 +104,42 @@
...
                 <div class="col-xs-12 col-md-4">
-                  <a href="news/announcing-schedule/" class="well news__tile">
+                  <a href="news/announcing-presentations-ticket-update/" class="well news__tile">
                     <h3 class="well__title">
-                      Announcing talk Schedule
+                      Announcing presentations!
                     </h3>
                     <p class="well__paragraph">
-                    Our website is now updated with the final schedule for
-                    the talks...
```
Comment (ericholscher) [inline docs/conf/au/2017/index.rst:143]: This is the part you won't have to do next year :)

File: docs/conf/au/2017/news/event-recap-feedback [hunk 2]
```diff
@@ -0,0 +1,143 @@
+:template: 2017/au-content.html
+
+.. post:: Dec 1, 2017
+   :tags: 2017, australia
+
+Write the Docs Day Australia 2017 - Recap
+=================================================
+
+A big thank you to all attending the 2017 and our first Australian Write the
+Docs Day  in Melbourne. We are extremely grateful to our sponsors, volunteers,
```
Comment (ericholscher) [inline docs/conf/au/2017/news/event-recap-feedback]: Some spacing issues here `Day  in`

File: docs/conf/au/2017/news/event-recap-feedback [hunk 3]
```diff
@@ -0,0 +1,143 @@
+:template: 2017/au-content.html
+
+.. post:: Dec 1, 2017
+   :tags: 2017, australia
+
+Write the Docs Day Australia 2017 - Recap
+=================================================
+
+A big thank you to all attending the 2017 and our first Australian Write the
```
Comment (ericholscher) [inline docs/conf/au/2017/news/event-recap-feedback]: A bit awkward. Perhaps:

    A big thank you to all attending the first Write the Docs Day in Melbourne.

File: docs/conf/au/2017/news/event-recap-feedback [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+Docs Day  in Melbourne. We are extremely grateful to our sponsors, volunteers,
+speakers and attendees. We loved the time, effort and passion put in by
+everyone for this event.
+
+Hopefully, this event met people’s expectations and provided an opportunity to
+learn, meet and  interact with other documentarians from a growing community.
```
Comment (ericholscher) [inline docs/conf/au/2017/news/event-recap-feedback]: I generally go for a more "everyone we talked to said this event met and exceeded their expectations!" vibe on these, but I'll leave it up to you. If you're going for humble, I can dig it :)

File: docs/conf/au/2017/news/event-recap-feedback [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+ and Adelaide. We will continue to organise events across Australia and will
+ love to get you involved, attending, speaking or volunteering at future events.
+
+You can get in touch with us via australia@writethedocs.org or subscribe for
+future events at: https://www.meetup.com/Write-The-Docs-Australia
+
+Slack
+----------
+
+The Write the Docs Slack has really grown this last year. Our Slack channels
```
Comment (ericholscher) [inline docs/conf/au/2017/news/event-recap-feedback]: Could use a link, to make it easier for folks to jump in

File: docs/conf/au/2017/news/event-recap-feedback [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+2018 conferences
+---------------------------
+
+It’s early days and we are working on a bigger and better version of Write
+the Docs Australia conference for 2018. Date, venue and call for proposals
+will be released early next year.
+
+In the meantime, save the dates for our US and EU next year:
+Portland conference, 6-8 May 2018
+Prague conference, 9-11 September 2018
```
Comment (ericholscher) [inline docs/conf/au/2017/news/event-recap-feedback]: Needs list formating (`*`)

**Review by ericholscher — COMMENTED** [review]
This is great :+1: A few small nits.

If this also will go out as an email to attendees, I'd include a feedback form link in it as well.

**Comment by ericholscher** [comment]
@swapnilogale pushed up some fixes.

**Comment by swapnilogale** [comment]
@ericholscher @plaindocs Happy for this to be merged. Thanks for reviewing.

**Comment by swapnilogale** [comment]
@plaindocs - can you hold on merging this PR? The videos and photos for the event are been uploaded today, so will include it in the update too.

**Comment by swapnilogale** [comment]
@plaindocs - Happy for this to be merged now. Links to photos and videos updated.

File: docs/conf/au/2017/news/event-recap-feedback.rst [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+------------------
+
+Writing (Half) Day
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The first half of the day was dedicated to a documentation sprint, with
+attendees contributing to one of the 5 open source projects that were pitched
+on the day. Attendees could choose from Dart, Kubernetes, ReactiveUI, WebPact
+or Cyrus mail.
+Sarah Maddox had blogged about the Doc sprint experience here:
```
Comment (plaindocs) [inline docs/conf/au/2017/news/event-recap-feedback.rst:33]: Probably makes sense to linkify this? And the tense reads odd to me

**Review by plaindocs — COMMENTED** [review]
I'll merge this in, and if you feel like updating as per that single comment do do.

**Pull request merged**